04LessonFour/in-class-project/index.html

```diff
@@ -1,10 +1,62 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title>The Daily Planet</title>
-  <link rel="stylesheet" href="css/style.css">
+    <title>Intel</title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
 
+    <header>
+        <h1>Intel</h1>
+        <nav>
+            <span><a href="#">i7 Processor</a></span>
+            <span><a href="#">Solid-State Drives</a></span>
+        </nav>
+    </header>
+
+    <main>
+

```

Comment:
this structure is great! consider giving more comments though, semantic comments that help you visualize what the different sections are.